fw/src/mppt/local/pando.hpp

```diff
@@ -35,7 +36,7 @@ class PandO final : public MPPT {
             #undef OUTPUT_CURRENT
         }
 
-        void step_algorithm(void) {
+        float step_algorithm(void) {
```

Comment:
Since this is modifying the API I might suggest making this the case across all instances of MPPT algorithm implementations.